asapdiscovery/simulation/scripts/MProSim_Sub.sh

```diff
@@ -3,15 +3,18 @@
 #BSUB -oo log_files/MProSim_Sub.out
 #BSUB -e log_files/MProSim_Sub.stderr
 #BSUB -n 1
+#BSUB -n 2 -W 2:00 -q gpuqueue -gpu 'num=1' -R 'span[hosts=1]'
```

Comment:
Remember that some of these flags are redundant! This line can just be
`-gpu 'num=1'`

Reply:
This is old I, need to push the new shell script, but yes - you're right!